app/views/subscriptions_management/update_address.html.erb

```diff
@@ -0,0 +1,59 @@
+<% content_for :title, 'Change your email address' %>
+
+<% content_for :head do %>
+  <meta name="robots" content="noindex, nofollow">
+<% end %>
+
+<% content_for :head_css do %>
+  <%= stylesheet_link_tag 'application-without-elements' %>
+<% end %>
+
+<%= render 'govuk_publishing_components/components/back_link', {
+  href: @back_url,
+} %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title', title: 'Change your email address' %>
+  </div>
+</div>
+
+<% if flash[:error] %>
+  <div class="grid-row responsive-bottom-margin">
+    <div class="column-two-thirds">
+      <%= render 'govuk_publishing_components/components/error_summary', {
+        title: 'We weren’t able to change your email address',
+        items: [
+          {
+            text: 'There’s a problem with your email address',
+            href: '#email-address-input',
+          }
+        ]
+      } %>
+    </div>
+  </div>
+<% end %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/govspeak', {
+      content: %{<p>Your current email address is #{@address}</p>},
+      rich_govspeak: true
+    } %>
+    <%= form_tag change_address_path, method: :post do %>
+      <%= render 'govuk_publishing_components/components/input', {
+        error_message: flash[:error],
+        id: 'email-address-input',
+        label: { text: 'What’s your new email address?' },
+        name: :new_address,
+        type: 'email',
+        value: @new_address,
+      } %>
+      <%= render 'govuk_component/button', {
+        text: 'Save',
+        margin_bottom: true,
+      } %>
+      or <a href="<%= list_subscriptions_path %>">cancel</a>
```

Comment:
can we use `@back_url` here instead? and also on https://github.com/alphagov/email-alert-frontend/pull/183/commits/35273141884a425cf1e71ff7b69462b99b9c5c16#diff-86da3a3fe96392fa8e55940bc662f3a3R56

Reply:
Done